contrib/firmware_packager/install_dell_bios_exe.py

```diff
@@ -101,6 +108,7 @@ def prompt_reboot():
     ARGS = parse_args()
     CLIENT = Fwupd.Client()
     check_exists(ARGS.exe)
+    early_exit_if_trusted_only()
```

Comment:
Weren't you going to try to do this automatically and then only exit if you couldn't?

Reply:
> Weren't you going to try to do this automatically and then only exit if you couldn't?

Yeah honestly if the `prop` is known, I was thinking to turn it off silently and restore it when finished. On the contrary what if it is unknown, between the listed options which is better? Ideas welcome.

1. let it be ultimately users will see the update failure
2. yes/no question to turn it off